src/packet_processor.rs

```diff
@@ -2,16 +2,13 @@ use super::game_state::block::BlockStateOperations;
 use super::game_state::patchwork::PatchworkStateOperations;
 use super::game_state::player::PlayerStateOperations;
 use super::messenger::MessengerOperations;
-use super::packet::{read, Packet};
+use super::packet::{read, translate};
 use super::packet_router;
 use std::collections::HashMap;
 use std::io::Cursor;
 use std::sync::mpsc::{Receiver, Sender};
 use uuid::Uuid;
 
-const ENTITY_ID_BLOCK_SIZE: i32 = 1000;
-const CHUNK_SIZE: i32 = 16;
```

Comment:
Not really sure where to put these. I think they're fine in the patchwork state for now and hopefully a clear place to put them will come up